src/tests/specs/workItemTableTest.spec.ts

```diff
@@ -49,10 +44,12 @@ describe('Work Item datatable list', () => {
 
   it('work item should show updated title when switching from flat to tree view', async() => {
     await planner.header.clickShowTree();
-    await planner.workItemList.clickWorkItem(c.workItemTitle1);
+    await planner.workItemList.overlay.untilHidden();    
+    await planner.workItemList.clickWorkItem(c.workItemTitle2);    
+    await planner.quickPreview.closeButton.untilDisplayed();
```

Comment:
This one can be moved to clickWorkItem method. Let's keep the tests clear of all (most of) the waits.

Reply:
done